docs/analytics/domain-proxy.md

```diff
@@ -100,9 +100,19 @@ This call should return a `200` response code. In the web app, confirm that Ampl
 After the proxy is working correctly, configure the SDK. Amplitude’s SDKs are open source code that you can change. The SDKs already have built-in options to send events to your defined server endpoint.
```

Comment:
@yuhao900914 sorry for the out of scope suggestion here, but this has always read weird to me. 

Can we rephrase this to something like: 


```suggestion
After the proxy is working correctly, configure your SDK. Amplitude’s SDKs are open source code that you can change and have built-in options to send events to your defined server endpoint.
```